src/dune/__main__.py

```diff
@@ -176,6 +176,15 @@
             elif args.version:
                 print("DUNE " + version_full())
 
+            elif args.upgrade:
+                dune_sys.docker.upgrade()
+
+            elif args.leap:
+                dune_sys.execute_cmd(['sh', 'bootstrap_leap.sh', args.leap])
+
+            elif args.cdt:
+                dune_sys.execute_cmd(['sh', 'bootstrap_cdt.sh', args.cdt])
+
```

Comment:
The script files won't be found as relative paths. We need to resolve the path to an absolute...

```py
# Find path for this file:                                                                                                                                            
MY_PATH = os.path.dirname(os.path.abspath(__file__))
```

and maybe 
```
dune_sys.execute_cmd(['sh', MY_PATH + '/bootstrap_leap.sh', args.leap])
```
or similar.

Reply:
After investigation it occurred that uploaded image ghcr.io/antelopeio/dune:1.0.0 was invalid - it did not contain `bootstrap_leap.sh` or `bootstrap_cdt.sh`.
I have uploaded new DUNE image. So to retest you might want to try `./dune --upgrade` - however version did not change, so if it doesn't download new version you need to `docker rm` dune images and then `./dune --upgrade`